crates/viewer/re_dataframe_ui/src/requested_object.rs

```diff
@@ -1,9 +1,9 @@
-use crossbeam_channel::{bounded, Receiver};
+use std::sync::mpsc::{sync_channel, Receiver};
 
 use re_viewer_context::{AsyncRuntimeHandle, WasmNotSend};
 
 /// A handle to an object that is requested asynchronously.
-#[derive(Debug, Clone)]
+#[derive(Debug)]
```

Comment:
Add a comment here explaining why it _shouldn't_ be `Clone`. Or even enforce it in a test.

Reply:
I'm not sure how to write that test. Do you have an example for that?

Reply:
You can use https://github.com/rerun-io/rerun/pull/9748

Reply:
Nice!